packages/vuepress-theme-odyssey/components/DocsCardGroup.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div
+    :class="{
+      'docs-card--group': true,
+      'is-2col': variant === '2col'
```

Comment:
Not possible to do this automatically based on number of children / content?

If not, could we maybe create a use-case based name here?

Reply:
Moved conversation to #728